boards/arm/mps3_an547/Kconfig.board

```
@@ -0,0 +1,6 @@
+# Copyright (c) 2021 Linaro Limited
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_MPS3_AN547
+	bool "Arm Cortex-M55 (Corstone SSE-300 w/Ethos-U55) on MPS3 (AN547)"
```

It might make more sense to refer to this as `Corstone-300` since that's the name that ARM uses?

```suggestion
	bool "Arm Cortex-M55 (Corstone-300, SSE-300 w/Ethos-U55) on MPS3 (AN547)"
```

---

Done.